components/admin/genders/SPECS/genders.spec

```diff
@@ -31,6 +31,10 @@ BuildRequires: python3-devel
 BuildRequires: byacc
```

Comment:
`%if 0%{?rhel_version} || 0%{?openEuler}` for byacc

Reply:
Improved it!